omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -130,7 +130,8 @@ def __init__(self, canvas, panel, page, crop, page_height):
                 elif shape['type'] == "Ellipse":
                     self.draw_ellipse(shape)
 
-    def get_rgb(self, color):
+    @staticmethod
+    def get_rgb(color):
```

Comment:
moving to @staticmethod means that this could be used everywhere and no need to have the same method line 424